packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,17 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCount: 0 }),
+  effects: {
+    refreshXoaStatus() {
+      this.state.checkXoaCount += 1
+    },
+  },
+  computed: {
+    xoaStatus: async ({ checkXoaCount }, props) => await checkXoa(),
```

Comment:
```suggestion
    xoaStatus:  ({ checkXoaCount }) => checkXoa(),
```

Reply:
By removing async/await, we are returning a `Promise` and not a string. This doesn't causes issues for get it in the UI ?

Reply:
Please read carefully the documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

Keep me posted if you still have question after 🙂